### PR TITLE
feat: copy fonts after install

### PR DESCRIPTION
**Pull request by Averethel**
Adds a `cli` to copy fonts so they can be self-hosted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,25 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 ## Usage
-```
-npm install @smg-automotive/example
-```
+
+1. Install the package
+   ```sh
+   npm install @smg-automotive/components
```
Comment (GutierrezGo) [inline README.md:10]: Thank you for adding the instructions.
Do we need to highlight that the `^` needs to be removed.

Comment (Averethel) [reply README.md:10]: I wouldn't. Modern npm shouldn't add them by default if all other dependencies are pinned. Worst case we ger renovate PR to pin

File: src/lib/cli/copy-fonts.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-console */
+import { destination, source } from './paths';
+
+const fs = require('fs-extra');
+
+export default ({ path: destinationRoot }: { path: string }) => {
+  const resolvedDestination = destination(destinationRoot);
+
+  console.info('copying fonts from to', resolvedDestination);
```
Comment (lkappeler) [inline src/lib/cli/copy-fonts.ts:10]: from where?

File: README.md [hunk 3]
```diff
@@ -4,9 +4,25 @@
...
+   ```
+2. run the setup script
+   ```sh
+   npx run components setup --path=<path to you public dir>
```
Comment (lkappeler) [inline README.md:14]: Don't we need to use `@smg-automotive/components` when running the package command with npx?

Comment (GutierrezGo) [inline README.md:14]:
```suggestion
   npx components setup --path=<path to you public dir>
```

**Review by lkappeler — APPROVED** [review]
Looks pretty solid to me. I think the setup is a bit overkill but since you added it let's keep it 🙂. Did not test it locally assume you did that. Nice job.

File: src/lib/cli/copy-fonts.ts (hunk 2 above)
Comment (Averethel) [reply src/lib/cli/copy-fonts.ts:10]: Ah `from` is a leftover, will remove

File: README.md (hunk 3 above)
Comment (Averethel) [reply README.md]: it depends. If we have it installed and listed in dependencies (which the setup assumes), then no.
If you would want to install the package then you'd need run it with the package:
```
npx -p @smg-automotive/components -c components setup
```

File: src/lib/cli/copy-fonts.ts (hunk 2 above)
Comment (Averethel) [inline src/lib/cli/copy-fonts.ts:10]:
```suggestion
  console.info('copying fonts from the package to', resolvedDestination);
```

**Pull request merged**